### DISCUSSION TITLE
Github Action Workflow Run on deployment

### DISCUSSION THREAD

**Discussion by judoole** [community/community#26137] [Actions]
<p>Hey!</p>
<p>I’m trying to start a workflow with <strong>Run</strong> on <strong>deployment</strong> , but it never starts. I can see from the <strong>environment</strong> tab that the event is received, and in the end <strong>Abandonded</strong> , probably because of timeout I guess. The other workflow, which is triggered on  <strong>push</strong> , runs fine.</p>
<p>To be more precise I am using <a href="https://hubot.github.com/" rel="nofollow noopener">Hubot</a> and <a href="https://github.com/atmos/hubot-deploy" rel="nofollow noopener">hubot deploy</a> for sending the deployment-events. </p>
<p>Any tips on why the workflow does not start?</p>

**Comment by judoole** [comment]
<p>Seemed to work when I removed the other workflow.</p>
<p>Is it so that there is only room for one workflow? The editor seems to think otherwise.</p>

**Comment by mcolyer** [comment]
<p><a class="mention" href="/judoole">@judoole</a> this is very odd, it should work with more than one workflow. I just tried it out on one of my own test repositories and it seems to work, so there must be a detail we’re missing. Can you share more details about the workflows or the events your sending?</p>
<p>If it contains private information we can move it to <a href="mailto:support@github.com">support@github.com</a> if that’s easier.</p>

**Comment by mcolyer** [comment]
<p>I just had one more thought, which branch is the workflow located on and which ref are you sending in the deployment event?</p>

**Comment by judoole** [comment, marked as answer]
<p>Sorry to have bothered you <a class="mention" href="/mcolyer">@mcolyer</a><br>
I just tested with a supersimple hello-worldish repository, and multiple Workflows works brilliantly.</p>
<p>This time I’m just testing by doing a curl to the deployments api. Might be that hubot-deploy or something else was askew with the other repository.</p>
<p>Thanks for taking the time to answer!</p>
